src/components/Error/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './Error';
```

Comment:
Consider explicitly naming the export for better understanding and easier debugging. For example: export { default as Error } from './Error';